@shared/api/external.ts

```diff
@@ -20,10 +20,10 @@ export const requestPublicKey = async (): Promise<{
 export const submitTransaction = async ({
   transactionXdr,
 }: LyraApiRequest): Promise<{
-  transactionStatus: string;
+  signedTransaction: string;
```

Comment:
since we're returning an actual transaction (instead of the response from Horizon like we were originally) let's change what this key is called